cloudamqp/resource_cloudamqp_notification.go

```diff
@@ -2,7 +2,7 @@ package cloudamqp
 
 import (
 	"errors"
-	"fmt"
+	"log"
```

Comment:
Just curious, why did this change? :)

Reply:
Refactoring of this part: https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/main/cloudamqp/resource_cloudamqp_notification.go#L98-L104
Removed the fmt.Errrof(...) and added log.Printf(...) on other places.